routes/api.php

```diff
@@ -76,22 +71,14 @@
 Route::post('/notifications/schedule', [NotificationController::class, 'createScheduleNotification']);
 Route::post('/notifications', [NotificationController::class, 'createNotification']);
 
-
-
-
-
-
-// Schedule routes
-Route::get('/schedules', [ScheduleController::class, 'get_all_schedules']);
-Route::post('/schedules/generate', [ScheduleController::class, 'generate_schedules']);
-Route::delete('/schedules/{schedule_id}', [ScheduleController::class, 'delete_schedule']);
-Route::delete('/schedules', [ScheduleController::class, 'delete_all_schedules']);
-Route::get('/schedules/day/{day}', [ScheduleController::class, 'filter_schedules_by_day']);
-Route::get('/schedules/search', [ScheduleController::class, 'search_schedules']);
-Route::get('/schedules/export', [ScheduleController::class, 'export_schedules_to_csv']);
-
-
 // Auth
 Route::post('/register', [RegisterController::class, 'register_user']);
 Route::post('/login', [LoginController::class, 'authenticate']);
-Route::post('/logout', [LoginController::class, 'logout']);
\ No newline at end of file
+Route::post('/logout', [LoginController::class, 'logout']);
+
+// Student enrollment routes
+Route::get('/courses/available', [StudentEnrollmentController::class, 'getAvailableCourses']);
```

Comment:
Lebih baik filter diquery imo
```suggestion
Route::get('/courses', [StudentEnrollmentController::class, 'getAvailableCourses']);
```

lalu nanti di model (kode di bawah belum tes)
```
class Model {
    public function namaMetode(Request $request){
       $course_filter = $request->q
       ....
   }
}